File: UniversityApplicant.API/UniversityApplicantAPI/Models/Applicant.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;
using UniversityApplicantAPI.Enums;

namespace UniversityApplicantAPI.Models
{
    public class Applicant
    {
        [Key]
        public Guid Id { get; set; }
        public string Fname { get; set; }
        public string Lname { get; set; }
        public string Email { get; set; }
        public DateTime BirthDate { get; set; }
        [NotMapped]
        public int? Age { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public string Address { get; set; }
        public eGenderType Gender { get; set; }
        public string PhoneNo { get; set; }
        public eMaritalStatus MaritalStatus { get; set; }
        public string SchoolName { get; set; }
        public int GraduationYear { get; set; }
        public eGrade? Grade { get; set; }
        public string ApplicantResume { get; set; }
    }
}

File: UniversityApplicant.Client/workspace/src/app/applicant.ts
export class Applicant {
    public Id: string;
    public Fname: string;
    public Lname: string;
    public Email: string;
    public BirthDate: Date;
    public Age: number;
    public Country: string;
    public City: string
    public Address: string;
    public Gender: eGenderType;
    public PhoneNo: string;
    public MaritalStatus: eMaritalStatus;
    public SchoolName: string;
    public GraduationYear: number;
    public Grade: eGrade;
    public ApplicantResume: string;
}

export enum eGenderType {
    Male = 1,
    Female = 2
}

export enum eMaritalStatus {
    Single = 1,
    Married = 2,
    Divorced = 3,
    Widowed = 4
}

export enum eGrade {
    Fair = 0,
    Good = 1,
    VeryGood = 3,
    Excellent = 4
}
